subprojects/dependency-management/src/test/groovy/org/gradle/api/internal/artifacts/configurations/DefaultConfigurationSpec.groovy

```diff
@@ -1880,30 +1884,30 @@ All Artifacts:
         _ * domainObjectContext.model >> RootScriptDomainObjectContext.INSTANCE
 
         def publishArtifactNotationParser = new PublishArtifactNotationParserFactory(
-            instantiator,
-            metaDataProvider,
-            TestFiles.resolver(),
-            TestFiles.taskDependencyFactory(),
+                instantiator,
```

Comment:
These whitespace changes should be reverted

Reply:
So I'm fairly up-to-date with `master` here and this is the format IntelliJ suggests for these lines.  

Didn't the automatic formatting conventions recently get worked out?  I'll update this, but it's annoying to not have the IDE be the arbiter of proper indentation.  If this can't be automatically enforced I don't think it's worth trying to enforce via code review, unless something is radically off-base.